openshift/template.yaml

```diff
@@ -0,0 +1,242 @@
+apiVersion: template.openshift.io/v1
+kind: Template
+metadata:
+  name: configuration-anomaly-detection-template
+parameters:
+- name: IMAGE_DIGEST
+  value: sha256:22d2a957d935e883f45addd35acc87450ce71ea2f94c33f2df70ff36b015486a
```

Comment:
consider removing `value` and adding `required: true` instead?

similar to https://github.com/openshift/aws-account-operator/blob/0dc92d1e582f8f381f9a212b2edc4725394e2cb6/hack/olm-registry/olm-artifacts-template.yaml#L11-L12

Reply:
nah, I want the usual dev to have the correct image (and hopefully the correct one).
currently this image cannot pull the correct credentials, so it's ok I think

Reply:
so are you planning to maintain the image in this template forever?

Reply:
I was hoping I could, is that ask too much?
I can remove if it is

Reply:
i don't think it's feasible. you also can't know the digest before the image is pushed. and that only happens after a merge. :chicken: :egg:

Reply:
i think that would be better, and also consistent with other operators.

Reply:
#60